src/sage/combinat/crystals/all.py

```diff
@@ -1,25 +1,14 @@
 r"""
 Crystals
 
-Introductory material
```

Comment:
Why is this an improvement ? you remove useful links.

Reply:
I restored the original.